src/antsibull/data/ansible-readme.rst

```diff
@@ -28,56 +28,49 @@ Design Principles
 Use Ansible
 ===========
 
-You can install a released version of Ansible via ``pip``, a package manager, or
-our `release repository <https://releases.ansible.com/ansible/>`_. See our
-`installation guide <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_ for details on installing Ansible
+You can install a released version of Ansible with ``pip`` or a package manager. See our
+`Installation guide <https://docs.ansible.com/ansible/latest/installation_guide/index.html>`_ for details on installing Ansible
 on a variety of platforms.
 
-Red Hat offers supported builds of `Ansible Engine <https://www.ansible.com/ansible-engine>`_.
-
-Power users and developers can run the ``devel`` branch, which has the latest
-features and fixes, directly. Although it is reasonably stable, you are more likely to encounter
-breaking changes when running the ``devel`` branch. We recommend getting involved
-in the Ansible community if you want to run the ``devel`` branch.
 
 Get Involved
 ============
 
-*  Read `Community Information <https://docs.ansible.com/ansible/latest/community>`_ for all kinds
-   of ways to contribute to and interact with the project, including mailing list information and how
-   to submit bug reports and code to Ansible.
+*  Read `Community Information <https://docs.ansible.com/ansible/latest/community>`_ for ways to contribute to 
+   and interact with the project, including mailing list information and how
+   to submit bug reports and code to Ansible or Ansible collections.
 *  Join a `Working Group <https://github.com/ansible/community/wiki>`_, an organized community
    devoted to a specific technology domain or platform.
-*  Submit a proposed code update through a pull request to the ``devel`` branch.
+*  Submit a proposed code update through a pull request to the ``devel`` branch for `ansible-core` or the main branch 
+   for a collection update.
```

Comment:
I'm not sure about this line, maybe we should take it out since it's encompassed by the link to the community website on how to contribute

Reply:
removed, thanks